src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Divide program logic
Large methods is very difficult to read and understand. So methods should be short, between 5-15 lines. And it’s important for you to remember that one method should be responsible for one task.

For example: “write to file” or “create report” - are a separate operations. So we should create separate methods for them.